ntbs-service/Pages/Notifications/Overview.cshtml.cs

```diff
@@ -99,5 +103,11 @@ public async Task GetAlertsAsync()
         {
             Alerts = await _alertService.GetAlertsForNotificationAsync(NotificationId, User);
         }
+        
+        public async Task<ContentResult> OnPostAuditPrintAsync()
+        {
+            await _auditService.AuditPrint(NotificationId, HttpContext.User.Username() ?? HttpContext.User.Identity.Name);

```

Comment:
Why would the `Username()` be `null`?

Reply:
When the user doesn't have an associated email: https://github.com/publichealthengland/ntbs_Beta/blob/fa782b843c048a123b069a44efca02678b3a7c4a/ntbs-service/Middleware/AuditGetRequestMiddleware.cs#L47

Reply:
In that case I think you should commonise the logic here with that in the `AuditGetRequestMiddleware`

Reply:
Done 👍